sphinx/source/recovery.rst

```diff
@@ -1,105 +1,122 @@
 Catastrophic Recovery
 =====================
 
-For unexpected reasons, all nodes in the network may crash while executing business transactions. In this catastrophic scenario, it is possible that the length of ledgers of each node may differ slightly since some transactions may not have yet been fully replicated.
+For unexpected reasons, a significant number [#crash]_ of CCF nodes may crash. In this catastrophic scenario, it is possible that the length of ledgers of each node may differ slightly since some transactions may not have yet been fully replicated.
```

Comment:
Crash may not be the best description here, I think unavailable or lost is more accurate, because it also covers partitioning from the network.